packages/create-block-app/create-block-app.js

```diff
@@ -75,52 +73,53 @@ const usage = commandLineUsage(helpSections);
   const resolvedBlockPath = path.resolve(untildify(folderPath));
 
   try {
-    fs.statSync(resolvedBlockPath);
+    await fs.stat(resolvedBlockPath);
     console.error(
       `${resolvedBlockPath} already exists, please specify another path!`,
     );
     process.exit();
   } catch {
-    // noop (we expect statSync to fail)
+    // noop (we expect stat to fail)
   }
 
   const tempExtractionDir = path.join(resolvedBlockPath, "tmp");
 
-  console.log("Downloading template...");
+  const templatePackageName = `block-template-${template}`;
```

Comment:
`template` is checked earlier on line 61 (`if (!availableTemplates.includes(template)) {...`)